components/consumers/dependency-track/main.go

```diff
@@ -164,6 +201,7 @@ func addOwnersTags(owners []string) error {
 }
 
 func uploadBOM(bom string, projectVersion string) (string, error) {
+	slog.Info(fmt.Sprintf("Uploading BOM to Dependency Track for project %s version %s", projectName, projectVersion))
```

Comment:
This is not a good use of slog. The logger is expected to receive contextual values as parameters, so that they can be used as filters in a json filtering. The correct use would be like this:
```golang
slog.Info("uploading BOM to Dependency Track, "projectName", projectName, "projectVersion", projectVersion)
```